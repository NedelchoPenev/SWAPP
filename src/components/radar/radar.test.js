import React from 'react';
import { shallow } from 'enzyme';

import Radar from './radar.component';

describe('Radar component', () => {
  it('should render Radar component', () => {
    expect(shallow(<Radar />)).toMatchSnapshot();
  });
});
