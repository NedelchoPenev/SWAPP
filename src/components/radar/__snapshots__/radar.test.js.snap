// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Radar component should render Radar component 1`] = `
<styled.div>
  <h2>
    Compared to Starship Class Max
  </h2>
  <styled.div>
    <f
      captions={
        Object {
          "cost": "Cost",
          "crew": "Crew",
          "hyperdriveRating": "HyperD Rat.",
          "maxAtmosphericSpeed": "Max Atm. Speed",
          "maxMLPerHour": "Max ML/h",
        }
      }
      data={
        Array [
          undefined,
        ]
      }
      options={
        Object {
          "axes": false,
          "dots": true,
          "scales": 5,
        }
      }
    />
  </styled.div>
</styled.div>
`;
